crypto/default/src/main/java/org/keycloak/crypto/def/BCCertificateUtilsProvider.java

```diff
@@ -212,6 +212,7 @@ private ContentSigner createSigner(PrivateKey privateKey) {
             }
             return signerBuilder.build(privateKey);
         } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
Is this really needed? Isn't the throwing of runtimeException enough?

If the RuntimeException is somewhere not logged due the fact that it is ignored, then maybe we should rather improve the place where the exception is incorrectly ignored without logging the details. As ideally, the exception stacktraces should be properly logged with the proper logger rather than to STDERR.

Reply:
this was a mistake. I did not want to add this :-)
But I found two places where the exception was not logged on debug level and added it there.